pkg/database/migrations.go

```diff
@@ -2038,6 +2038,27 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 				return nil
 			},
 		},
+		{
+			ID: "00090-AddSMSSigningKeys",
+			Migrate: func(tx *gorm.DB) error {
+				return multiExec(tx,
+					`CREATE TABLE sms_signing_keys (
+						id BIGSERIAL,
+						created_at TIMESTAMP WITH TIME ZONE,
+						updated_at TIMESTAMP WITH TIME ZONE,
+						deleted_at TIMESTAMP WITH TIME ZONE,
+						realm_id INTEGER,
+						key_id TEXT,
+						active BOOLEAN,
```

Comment:
We should create an index that ensures only one value for `active` can exist in the scope of a `realm_id` (there's some other tables that have this, it's a conditional index)

Reply:
please double check my index statement